src/createAdder.js

```diff
@@ -19,10 +19,18 @@
  *
  * @param {number} initialValue
  *
- * @return {number}
+ * @return {function(...[*]): (*|number)}
  */
 function createAdder(initialValue = 0) {
-  // write code here
+  let result = initialValue;
+
+  return function(...arg) {
```

Comment:
аргументов больше одного, значит множественное число
```suggestion
  return function(...args) {
```